tests/integration/api_v3/http/spec.test.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { api } = require('../../../helpers/api');
+
+const {
+	badRequestSchema,
+} = require('../../../schemas/httpGenerics.schema');
+
+const {
+	specResponseSchema,
+} = require('../../../schemas/api_v3/spec.schema');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const baseUrlV3 = `${baseUrl}/api/v3`;
+const endpoint = `${baseUrlV3}/spec`;
+
+describe('Swagger spec API', () => {
+	it('return swagger specifications -> 200 OK', async () => {
+		const response = await api.get(endpoint);
+		expect(response).toMap(specResponseSchema);
```

Comment:
The test case can be expanded to be a bit more exhaustive? Alternatively, the schema can also be adjusted to make the checks a little more strict. We can discuss this offline.